deployment/hasura/migrations/AerieMerlin/23_add_units_to_activity_types/down.sql

```diff
@@ -0,0 +1,11 @@
+comment on column activity_type.parameter_units is null;
+comment on column activity_type.computed_attribute_units is null;
```

Comment:
When talking with @duranb he brought up this new column and asked what was the reason for it? Could we instead just add all this data to the existing `parameter`? It would make the API much easier to use.

Reply:
It's added in this way to keep it consistent with computed attributes and resource types.

We don't store computed attributes in the same way, only their schema so I don't think it makes sense to alter that object, same with resource types.

Maybe we should huddle to discuss further?

Reply:
That reasoning makes sense. Sure ping me and we can huddle!